hive-plugins/README.rst

```diff
@@ -0,0 +1,127 @@
+============================================
+Hive Batch Source and Sink Plugin Collection
+============================================
+
+Introduction
+============
+
+This project is a collection of Hive Batch source and sink plugins. Following is list of plugins that are currently available:
+
+- Hive Batch Source,
+- Hive Batch Sink,
+
+Getting Started
+===============
+
+Following are instructions to build and deploy hydrator Hive plugins.
+
+Prerequisites
+-------------
+
+To use plugins, you must have CDAP version 3.2.1 or later. You can download CDAP Standalone that includes Hydrator `here <http://cask.co/downloads>`__
+
+Build Plugins
+-------------
+
+You can get started with CDAP-plugins by building directly from the latest source code::
+
+  git clone https://github.com/caskdata/hydrator-plugins.git
+  cd hive-plugins
```

Comment:
We don't need this cd hive-plugins. We should always build form top level. mvn command won't work otherwise.